packages/@uppy/provider-views/src/ProviderView/ProviderView.jsx

```diff
@@ -237,6 +238,26 @@ export default class ProviderView extends View {
     }
   }
 
+  async loadAllPages () {
```

Comment:
would be nice if we could de-duplicate this (e.g. make a reusable function that this method and `handleScroll` will use)